build.gradle

```diff
@@ -116,7 +116,8 @@ dependencies {
     implementation group:'com.fasterxml.jackson.datatype', name: 'jackson-datatype-jsr310', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.datatype', name: 'jackson-datatype-hibernate5', version: versions.fasterXmlJackson
     implementation group:'com.fasterxml.jackson.core', name: 'jackson-annotations', version: versions.fasterXmlJackson
-    implementation group:'com.fasterxml.jackson.core', name: 'jackson-databind', version: versions.fasterXmlJackson
+    implementation group:'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.13.2.1'
```

Comment:
I am not sure this version can be downloadable

Reply:
This was the advisory for the vulnerability: https://github.com/FasterXML/jackson-databind/issues/2816
The project compiled fine after the changes in this PR